tools/detect_targets.py

```diff
@@ -92,6 +99,24 @@ def main():
         traceback.print_exc(file=sys.stdout)
         print "[ERROR] %s" % str(exc)
         sys.exit(1)
+        
+def get_interface_version(mount_point):
+    """ Function returns interface version from the target mounted on the specified mount point
+            Example of mount_point:
+                mount_point = mut['disk']
```

Comment:
I'm still not clear to me why I need an example of what to do with the parameter here. Are you suggesting that a user might be able to acquire a `mount_point` with the example code? If so, could you state that?

Reply:
Pushed a new commit with a bit more clarity, apologies for the confusion ;)